condor_watch_q.py

```diff
@@ -260,11 +260,13 @@ def watch_q(
                     file=sys.stderr,
                 )
 
-            msg = table_by(tracker.clusters, groupby, abbreviate_path_components=abbreviate_path_components)
+            summary, msg = table_by(tracker.clusters, groupby, abbreviate_path_components=abbreviate_path_components)
             msg = msg.splitlines()
             msg += ["", "Updated at {}".format(now)]
             msg = "\n".join(msg)
-
+            srm = "\n" + str(summary["TOTAL"]) + " jobs; " + str(summary["COMPLETED"]) + " completed, " + str(summary["REMOVED"]) + " removed, " + str(summary["IDLE"]) + " idle, " + str(summary["RUNNING"]) + " running, " + str(summary["HELD"]) + " held, " + str(summary["SUSPENDED"]) + " suspended"
```

Comment:
Let's do this in a single `str.format` call (https://docs.python.org/3/library/stdtypes.html#str.format)